src/geant4/app/src/DagSolidTally.cc

```diff
@@ -0,0 +1,28 @@
+#include "DagSolidTally.hh"
+#include "uwuw.hpp"
+
+// loads the tallies from the file
+std::map<std::string, pyne::Tally> load_uwuw_tallies(std::string filepath) {
+
+  bool end = false;
+  std::map<std::string, pyne::Tally> tally_library;
+  int i = 0;
+
+  std::cout << filepath << std::endl;
+  while (!end) {
+    pyne::Tally tally; // from file
+    std::cout << i << std::endl;
```

Comment:
I would add a label to this `std::cout` so the user knows what is being counted.